ui/app/src/utils/variables.ts

```diff
@@ -12,17 +12,38 @@
 // limitations under the License.
 
 import { ExternalVariableDefinition } from '@perses-dev/dashboards';
-import { VariableResource } from '@perses-dev/core';
+import { Variable } from '@perses-dev/core';
 import { ExternalVariableSource } from '../model/variables';
 
+export function buildProjectVariableDefinition(projectName: string, variables: Variable[]): ExternalVariableDefinition {
+  return {
+    editLink: `/projects/${projectName}/variables`,
```

Comment:
This is great, love how it links directly to the correct project’s variable tab and describes how overriding works